### DISCUSSION TITLE
BOM Generation @cyclonedx/cyclonedx-npm

### DISCUSSION THREAD

**Discussion by khaledgithubwl** [CycloneDX/cyclonedx-node-module#345] [Q&A] [answered]
Hello, 
I have a question regarding the content of the bom.
when using @cyclonedx/bom@3.0.2, we don't find the same result when using @cyclonedx/cyclonedx-npm 
Detected components and vulnerabilities are drastically higher when using @cyclonedx/cyclonedx-npm 
Does the new bom show all the subdependencies which explains this difference?

Best Regards,

**Comment by jkowalleck** [comment, marked as answer]
> Does the new bom show all the subdependencies which explains this difference?
 
exactly.
